### PR TITLE
Push plugin assets to wp.org plugin SVN

### PR DESCRIPTION
**Pull request by gonzomir**

### CODE REVIEW

**Comment by alexbhr** [comment]
You added a new action to push to the repo with every commit to master. That should not happen - the only time we should be pushing to the SVN repo is when a version is released and tagged. Its okay if the only changed thin is a readme file - most WP hosting providers update plugins pretty much automatically nowadays, so I don't see much of an issue with that. Please remove the new action and LGTM.

**Comment by gonzomir** [comment]
@alexbhr The action doesn't push to SVN on every commit. It pushes ONLY if the changes to be pushed contain ONLY changes to `plugin.txt` or assets.

> This Action commits any readme and WordPress.org-specific assets changes in your specified branch to the WordPress.org plugin repository if no other changes have been made since the last deployment to WordPress.org.
> https://github.com/10up/action-wordpress-plugin-asset-update

It's not OK to create releases for changes in assets or plugin file, think of them as plugin's website - one doesn't create software release when updating software's website.

WP hosting providers do not (and should not) update plugins automatically. WordPress has autommatic update functionality, but website owners, especially on commerce websites, are hesitant on enabling them because they may cause problems when breaking changes are introduced.

File: .github/workflows/plugin-assets.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Plugin assets/readme update
+on:
+  push:
+    branches:
+      - master
```
Comment (alexbhr) [inline .github/workflows/plugin-assets.yml:5]: @gonzomir this action will only run on every push to master. If you want it to make it run exclusively when a specific file is changed, please try something like:

```yaml
on:
  push:
    branches:
      - master
    paths:
      - 'readme.txt'
      - 'assets/**'
      - 'etc'

```

**Review by alexbhr — CHANGES_REQUESTED** [review]
I hope my comment above makes sense - otherwise, this action will be triggered with every push to master, regardless if it needs to push the assets or not.

**Comment by gonzomir** [comment]
@alexbhr I removed the action for publishing assets to WP.org, I'll leave merging and publishing the changes to you. Please, also review changes to `readme.txt`.

I want to once again note that creating plugin releases just for updates to assets and the readme is a bad practice and will lead to users being confused of new version of the plugin that has no real changes.

**Comment by atanasantonov** [comment]
Hi @alexbhr I've made minor changes in readmes most of all. Placed a link to changelog.txt in readme.txt and removed links to images to follow [WordPress plugin readme file standard](https://wordpress.org/plugins/readme.txt). Replaced the image in README.md with the new banner and deleted the unused images in assets folder. And actual tested up to version.

**Comment by alexbhr** [comment]
Excellent @atanasantonov - I have pushed the release, lets see if it works ;)

**Comment by atanasantonov** [comment]
I've added a tag and pushed it to trigger the release action. With a little fix in between ;)

**Pull request merged**